content/validation/before/domains.md

```diff
@@ -0,0 +1,39 @@
+Data plays a crucial role in the process of training any ML model. It is something from which the model learns to solve a given task and therefore care needs to be taken with its handling:
+
+1. By making sure that a data set is relevant to the problem and no deviation from expectations is introduced. This makes sure that the model doesn't incorporate undesired fluctuations or features.
+1. By performing a proper preprocessing of the data set so that the training step goes smoothly.
```

Comment:
This should be 2 instead of 1. In general I think you could frame this a little more, maybe in the intro sentence say something like 'there are two main considerations when collecting and preparing data for an ML task

Reply:
This is actually a feature, one can just write "1." all the time in numbered lists and it will render correctly as 1., 2., 3. etc.